src/silo/preprocessing/preprocessor.test.cpp

```diff
@@ -83,6 +83,29 @@ const Scenario NDJSON_WITH_SQL_KEYWORD_AS_FIELD = {
    )
 };
 
+const Scenario NDJSON_WITH_NUMERIC_NAMES = {
+   .input_directory = "testBaseData/numericNames/",
+   .expected_sequence_count = 2,
+   .query = R"(
+      {
+         "action": {
+            "type": "Aggregated",
```

Comment:
Is this a relevant query for this test case? Let's maybe do a `Fasta` action instead?

Reply:
The FastaAligned action cannot filter for the numeric metadata field

Reply:
Ah, I overlooked that there is also a numeric metadata field :+1: